tsconfig.json

```diff
@@ -16,7 +16,8 @@
 
     // Strict mode, but allow implicit any.
     "strict": true,
-    "noImplicitAny": false
+    "noImplicitAny": false,
+    "skipLibCheck": true
```

Comment:
This is a temporary fix; TypeScript 4.x does not like some constructs in Octokit. If we update TypeScript to 5.x, we have to update `jest`-related packages and `@automattic/eslint-plugin-wpvip`. The latter generates too many errors to be fixed in this PR.